docs/src/xhtml/components/forms/custom-select.xhtml

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Custom Select</title>
+	<object data="tabs.xhtml"></object>
+	<script>
+		$(document).ready(function() {
+			$(".user-container div").click(function(e){
+				var name = $(e.currentTarget).attr("name");
+				ts.ui.get("#myAside", aside => aside.close());
+				$("#mySelect").val(name);
+				ts.ui.Notification.success(name);
+			});
+		});
+	</script>
+	<style>
+		.user-container div {
+			margin: 11px;
+			cursor: pointer;
+		}
+	</style>
+</head>
+	<body class="nosticky">
+		<header data-ts="TopBar"></header>
+		<main data-ts="Main">
+			<article>
+				<h1>Custom Select</h1>
+				<section class="desc">
+					<p>You can have a chance to make your own select, if you set the attribute <att>data-ts.custom="true"</att></p>
```

Comment:
maybe write something like 'you can handle the `onclick` action on a select input element' to make it more clear from the start